### PR TITLE
how-to/deploy: add a description to the environment configuration

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
This PR adds a description to the environment configuration.

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1856

### Which version does your change affect? 
dev, v2.1, v3.0

### CODE REVIEW

**Comment by lilin90** [comment]
@zyh-hust PTAL

File: dev/how-to/deploy/hardware-recommendations.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ You can deploy and run TiDB on the 64-bit generic hardware server platform in th
 >
 > - In the test environment, the TiDB and PD can be deployed on the same server.
 > - For performance-related test, do not use low-performance storage and network hardware configuration, in order to guarantee the correctness of the test result.
+> - TiKV requires NVMe SSD disks to ensure faster reads and writes.
```
Comment (yikeke) [inline dev/how-to/deploy/hardware-recommendations.md]: The original Chinese sentence has been updated. Plz update this accordingly.

File: dev/how-to/deploy/hardware-recommendations.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ You can deploy and run TiDB on the 64-bit generic hardware server platform in th
 >
 > - In the test environment, the TiDB and PD can be deployed on the same server.
```
Comment (lilin90) [inline dev/how-to/deploy/hardware-recommendations.md]:
```suggestion
> - In the test environment, the TiDB server and the PD server can be deployed on the same server.
```

File: dev/how-to/deploy/hardware-recommendations.md [hunk 3]
```diff
@@ -42,6 +42,7 @@ You can deploy and run TiDB on the 64-bit generic hardware server platform in th
...
 > - For performance-related test, do not use low-performance storage and network hardware configuration, in order to guarantee the correctness of the test result.
+> - NVMe SSD disks is recommended for TiKV to ensure faster reads and writes.
```
Comment (lilin90) [inline dev/how-to/deploy/hardware-recommendations.md]:
```suggestion
> - For the TiKV server, it is recommended to use NVMe SSDs to ensure faster reads and writes.
```

File: dev/how-to/deploy/hardware-recommendations.md (hunk 2 above)
Comment (yikeke) [reply dev/how-to/deploy/hardware-recommendations.md]: > 

是 TiDB 和 PD 实例可以部署到同一台服务器上？

Comment (lilin90) [reply dev/how-to/deploy/hardware-recommendations.md]: @yikeke Yes.

**Comment by lilin90** [comment]
@anotherrachel It seems that `NVMe SSDs` is more commonly used.

**Comment by anotherrachel** [comment]
> @anotherrachel It seems that `NVMe SSDs` is more commonly used.

Apparently so... Solid-state drive also called solid-state disk, I'll update it. Thanks.

**Pull request merged**